test/init_orgs_test.go

```diff
@@ -118,4 +118,44 @@ github.com/golang-migrate/migrate
 	// the environment takes a long time, it is bundled together here to speed up
 	// the tests
 	s.testSQL()
+
+	client, err := newSupersetClient()
+	req.NoError(err)
+
+	// Test the list of dashboards created in superset
+	s.T().Run("dashboard-list", func(t *testing.T) {
+		req := require.New(t)
+
+		links, err := client.dashboards()
+		req.NoError(err)
+
+		s.Equal([]string{
+			`<a href="/superset/dashboard/1/">Overview</a>`,
+			`<a href="/superset/dashboard/2/">Welcome</a>`,
+			`<a href="/superset/dashboard/3/">Collaboration</a>`,
+		}, links)
+	})
+
+	// Test gitbase queries through superset
+	s.T().Run("superset-gitbase", func(t *testing.T) {
+		req := require.New(t)
+
+		rows, err := client.gitbase("select * from repositories")
+		req.NoError(err)
+
+		s.Equal([]map[string]interface{}{
+			{"repository_id": "github.com/golang-migrate/migrate"},
+		}, rows)
+	})
+
+	// Test metadata queries through superset
+	s.T().Run("superset-metadata", func(t *testing.T) {
+		req := require.New(t)
+
+		rows, err := client.metadata("select * from organizations")
+		req.NoError(err)
+		req.Len(rows, 1)
+
+		s.Equal("golang-migrate", rows[0]["login"])
```

Comment:
Sorry if already discussed and I missed it, but: can we rely on this org?

Reply:
This was a conversation in the design doc between @smacker and me. The plan was to use golang-migrate for the initial milestone, and then later without rushing create a new one from scratch, with the minimal resources needed to test both sourced-ce and ghsync.
Maybe I should have opened an issue for this beforehand.